option.go

```diff
@@ -35,12 +35,18 @@ var (
 		WithPolicyErrRetryInterval("1m"),
 		WithPubkeyErrRetryInterval("1m"),
 		WithJwkErrRetryInterval("1m"),
-		WithATEnableMTLSCertificateBoundAccessToken(true),
-		WithATProcessorClientCertificateGoBackSeconds("1h"),
-		WithATProcessorClientCertificateOffsetSeconds("1h"),
+		// WithNewATProcessorParam(true, "1h", "1h"), // TODO: design how to have default, user cannot overwrite this one
```

Comment:
I think the user can override it to execute `WithNewATProcessorParam`.

Reply:
continue in https://github.com/yahoojapan/athenz-authorizer/pull/59#discussion_r402141793